index.rst

```diff
@@ -25,6 +25,7 @@ Contents of this documentation
    docs/ORM
    docs/webservice
    docs/editing
+   docs/entity-editor
```

Comment:
I think this is fine for now, and we can reorganize accordingly later on when we have more aspects of the codebase documented